UnitySDK/Assets/ML-Agents/Scripts/Brain.cs

```diff
@@ -18,21 +19,32 @@ public abstract class Brain : ScriptableObject
     {
         [SerializeField] public BrainParameters brainParameters;
 
-        protected Dictionary<Agent, AgentInfo> m_AgentInfos =
-            new Dictionary<Agent, AgentInfo>(1024);
+        protected List<Agent> m_AgentInfos = new List<Agent>(1024);
```

Comment:
This feels like a bad name now since it no longer has anything to do with AgentInfo

Reply:
fixed